pages/pydata.md

```diff
@@ -0,0 +1,30 @@
+title: PyData Track
+---
+
+This year, for the first time, the teams at PyCon and [PyData London](http://london.pydata.org/) are joining forces to deliver a data-science focussed track at the conference.
+***
+
+## What we are looking for
+This track is mainly focussed on looking for speakers who wish to share their experiences of using Python in data-science. Examples such as:
+
+  * How existing Python packages helped you to achieve your project goals:
+    * Data preparation
+    * Statistical analysis
+    * Machine learning
+    * Data visualisation
+  * Have you been working on developing your own Python applications for data science?
+    * Even if these are a _work-in-progress_, this is a great forum to present these!
+    
+The time allocated for each talk is 30 minutes. If you are happy to receive any questions from the audience (this is not compulsory), then please allow time for these.
```

Comment:
I could have sworn it was 25 minutes, but I could be wrong. I need to double check.